Sources/Stack.swift

```diff
@@ -13,30 +13,29 @@ public protocol Stack: class, Stackable {
 
 extension Stack {
     public var isHidden: Bool {
-        return (self.thingsToStack.filter({ !$0.isHidden }).count == 0)
+        return self.thingsToStack.allSatisfy { $0.isHidden }
     }
     
     func visibleThingsToStack() -> [Stackable] {
         return self.thingsToStack.filter({ !$0.isHidden })
     }
     
     fileprivate func viewsToLayout() -> [UIView] {
-        var views: [UIView] = []
-        let thingsToStack = self.visibleThingsToStack()
-        for i in 0..<thingsToStack.count {
-            let stackable = thingsToStack[i]
-            if let stack = stackable as? Stack {
-                let innerViews = stack.viewsToLayout()
-                views.append(contentsOf: innerViews)
-            } else {
-                if let view = stackable as? UIView {
-                    views.append(view)
-                } else if let fixedSizeStackable = stackable as? FixedSizeStackable {
-                    views.append(fixedSizeStackable.view)
+        return self
```

Comment:
Could prob drop the `self` save the line :)

Reply:
Sure thing :)